README.md

```diff
@@ -1,9 +1,43 @@
-# Alibeez-to-google-calendar-connector
+# Alibeez-to-Google-Calendar Connector
 
-Sync leaves from Alibeez to your Google Agenda account
+Copies leaves from Alibeez to Google Calendar.
 
-## Local env setup
+## How it works
 
- - Copy `.env.example` to `.env`, some values are required. You can ask `dreamlab@zenika.com` for the credentials
- - Run `npm install` to install dependencies
- - Run `npm start` to run the app
+The program needs permission from users to write to their calendars. Therefore,
+the work of the program is divided into two phases: an initialization flow where
+the program collects the permission of a user, and an update flow where leaves
+for all users are sync'ed up.
+
+### Initialization flow
+
+- User hits `/` with their browser, where they are invited to give permission to
+  the program to make changes to their calendars.
+- If the user proceeds, the program completes the OAuth 2 dance and acquires
+  some user information, including the necessary tokens to call the Calendar
+  API.
+- The program pulls the active future leaves of the user from Alibeez and copies
+  them over to Calendar.
+
+### Update flow
+
+- At regular intervals, the program pulls all leaves that have been updated
+  since the previous update flow.
+- For leaves for which the user has given permission (i.e. tokens are known) the
+  program either adds it to Calendar or removes it from Calendar, depending on
+  its status.
+
+## How to setup the project for local development
```

Comment:
```suggestion
## How to setup the project for local development?
```

Reply:
It's phrase like a question but not really a question.